Makefile.build

```diff
@@ -572,7 +572,7 @@ VALIDOPTS=$(if $(VERBOSE),,-silent) -o -m -coqlib .
 
 validate: $(CHICKEN) | $(ALLVO:.$(VO)=.vo)
 	$(SHOW)'COQCHK <theories & plugins>'
-	$(HIDE)$(CHICKEN) $(VALIDOPTS) $(ALLMODS)
+	$(HIDE)$(CHICKEN) $(VALIDOPTS) $(ALLVO)
```

Comment:
Note that CI has it own script in the gitlab.yml

Reply:
Thanks! Why doesn't it use `validate`?

Reply:
Because the files are in _install_ci/...